records_mover/records/targets/data_url.py

```diff
@@ -48,8 +48,11 @@ def move_from_dataframes_source(self,
                 move_from_dataframes_source(dfs_source=dfs_source,
                                             processing_instructions=processing_instructions)
 
-    def can_load_direct(self, scheme: str) -> bool:
-        return scheme == self.output_loc.scheme
+    def can_move_directly_from_scheme(self, scheme: str) -> bool:
+        # Currently all means of copying between different schemes
+        # involve streaming data down to Records Mover from the source
+        # and then back up to the target.
+        return False
```

Comment:
The integration tests pass fine here, so I don't think the implementation here has any effect currently - but I wanted it to be correct in case the move() logic ends up depending on it, and also so that the intended semantics are clear.

Note that arguably this could be set to 'true' for file -> file URL operations, since nothing leaves the box.  I might extend it to do that someday when that's relevant to optimizing a move() path.

Reply:
I honestly prefer the empty state to be False rather than True, mainly to avoid side effects.... 👍